client/main.lua

```diff
@@ -1007,3 +1007,733 @@ end)
 AddEventHandler('esx:onPlayerSpawn', function(spawn)
 	isDead = false
 end)
+local HasAlreadyEnteredMarker, LastZone = false, nil
```

Comment:
All these additions will cause breaking changes on the script.

Reply:
Removed lines 1007 to 1010, duplicate lines.